corehq/apps/userreports/models.py

```diff
@@ -395,7 +397,7 @@ def map_col(column):
                     'XFormInstance': _('Forms'),
                     'CommCareCase': _('Cases')
                 }.get(self.config.referenced_doc_type, "Layer"),
-                'columns': filter(None, [map_col(col) for col in self.columns])
+                'columns': ifilter(None, [map_col(col) for col in self.columns])
```

Comment:
`futurize` would convert this to a list comprehension. I think that should be our standard approach.